build-support/bin/release.sh

```diff
@@ -490,7 +552,7 @@ function usage() {
   echo "       and can be installed in an ephemeral virtualenv."
   echo " -l  Lists all pantsbuild packages that this script releases."
   echo " -o  Lists all pantsbuild package owners."
-  echo " -e  Check that native engine binaries are deployed for this release."
+  echo " -e  Check that wheels are prebuilt for this release."
```

Comment:
Do you think that you could add a comment somewhere to explain what process generally prebuilds the wheels (tagged travis builds? all travis builds?) and what consumes them (the release script)? Will this mean that a release tag must be pushed (and be past the relevant stage in travis) before the release can proceed? Fine either way, just hoping to understand.

Reply:
And, I just realized this breaks down for stable branch cherry-picks. I'll update the release condition to include any https://github.com/pantsbuild/pants branch.